components/Admin/Settings/JobRecommendationModal.tsx

```
@@ -0,0 +1,295 @@
+  useEffect(() => {
+    if (!searchTerm.trim()) {
+      setFilteredJobs(jobs);
+      return;
+    }
+
+    const lowerCaseSearch = searchTerm.toLowerCase();
+    const filtered = jobs.filter(
+      job => 
+        job.title.toLowerCase().includes(lowerCaseSearch) ||
+        job.company.toLowerCase().includes(lowerCaseSearch) ||
+        (job.location && job.location.toLowerCase().includes(lowerCaseSearch))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)